<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duck Decryptor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/duck.ico</normaloff>:/icons/duck.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setKeyButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>560</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>560</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>550</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="duckCppRB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DuckCpp</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="duckSharpRB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DuckSharp</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
